x-pack/plugins/security_solution/cypress/e2e/detection_alerts/alert_table_action_column.cy.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../objects/rule';
+import {
+  CLOSE_OVERLAY,
+  OVERLAY_CONTAINER,
+  SESSION_VIEWER_BUTTON,
+  ANALYZER_VIEWER_BUTTON,
+} from '../../screens/alerts';
+import { createRule } from '../../tasks/api_calls/rules';
+import { cleanKibana } from '../../tasks/common';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import { esArchiverLoad } from '../../tasks/es_archiver';
+import { login, visit } from '../../tasks/login';
+import { ALERTS_URL } from '../../urls/navigation';
+
+describe('Alerts Table Action column', { testIsolation: false }, () => {
+  before(() => {
+    cleanKibana();
+    esArchiverLoad('process_ancestry');
+    login();
+    createRule(getNewRule());
+    visit(ALERTS_URL);
+    waitForAlertsToPopulate();
+  });
+
+  it('session viewer button and session viewer should be visible', () => {
+    cy.get(SESSION_VIEWER_BUTTON).should('be.visible');
+    cy.get(SESSION_VIEWER_BUTTON).eq(0).trigger('click');
+    cy.get(OVERLAY_CONTAINER).should('be.visible');
+    cy.get(CLOSE_OVERLAY).trigger('click');
```

Comment:
Wrap it into a task method and move it to an after each hook.

Reply:
@MadameSheema Since `testIsolation` is false in this tests, we are making sure that each test do their own cleanup. So it was intentionally not put in after each because each test will not be have same cleanup as more test will be added to this.